core/arch/arm/include/mm/core_mmu_arch.h

```diff
@@ -210,6 +210,11 @@ static inline unsigned int core_mmu_get_va_width(void)
 	}
 	return 32;
 }
+
+void tlbi_mva_range(vaddr_t va, size_t len, size_t granule);
```

Comment:
Prefer static inline functions:

```
static inline void tlbi_va_range_asid(vaddr_t va, size_t len, size_t granule, uint32_t asid)
{
	tlbi_mva_range_asid(va, len, granule, asid);
}

static inline void tlbi_va_range(vaddr_t va, size_t len, size_t granule)
{
	tlbi_mva_range(va, len, granule);
}
```

Since core_mmu.h does include _mm/core_mmu_arch.h_, everything will be in place for callers.